### ISSUE TITLE
'organization' usage intent

### ISSUE THREAD

**Issue by peterpramb** [elastic/ecs#726]
From the [description](https://github.com/elastic/ecs/blob/master/schemas/organization.yml) it is (for me) not entirely clear what the usage intent of `organization` is:

Is it safe to be used by "end users" to label their events according to the originator (e.g. a business unit in a multi-tenant setup), or might it already be pre-populated to identify the event "origin" (e.g. a vendor's company name of whatever producing the event)?

**Comment by peterpramb** [comment]
Or maybe support both use cases by making it reusable, like `provider.organization` and `vendor.organization`?

**Comment by webmat** [comment]
Good point, yeah the purpose of `organization` is for you to fill this with your organization's information. As an operator, I'd be surprised (and annoyed) if a random Filebeat module started populating this with vendor information :-)

You're also right that we could also reuse the field set in a few places. When doing that, then it would make sense to fill this with vendor information and the like. Places where I could see this being useful is when analyzing things like certificates & the like, where we have org names, organizational units & other typical attributes.

The field set is also still pretty bare, we haven't had time to work on it more since 1.0. If you have ideas on additional fields you'd like to see in there, you're welcome to chime in on this :-)

**Comment by peterpramb** [comment]
Looks like Filebeat already "reuses" the `organization` field set in `as.organization` (`client.as.*`, `destination.as.*`, `server.as.*`, `source.as.*`).

**Comment by peterpramb** [comment]
From a logical perspective a hierarchical structure would be great, for example:
```
organization
└── (business) unit
    └── division
        └── department
            └── team
                └── member
```
But as organization structures differ, a flat structure is likely the way to go:
```
organization.id
organization.name

organization.unit.id
organization.unit.name

organization.division.id
organization.division.name

organization.department.id
organization.department.name

organization.team.id
organization.team.name

organization.member.id
organization.member.name
```
Using this for certificates though will provide a naming problem, as the meaning of `organizational unit` in a certificate context is specific to that organization and can refer to anything from the list above to something completely different.

Not sure if this makes sense...
